cves/CVE-2013-4315.yml

```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In versions 1.4.x and 1.5.x of Django, an oversight in ssi template tags granted
```

Comment:
Could you explain a bit more what is this tag?

Reply:
The {% ssi %} tag takes a file path for a template and includes the files contents and can parse the file to create a template.